content-src/asrouter/templates/FullPageInterrupt/_FullPageInterrupt.scss

```diff
@@ -0,0 +1,213 @@
+  .brand-logo {
+    background-image: url('chrome://branding/content/about-logo.png');
+    margin: 20px 10px 10px 20px;
+    padding-bottom: 50px;
+  }
+
+  .welcome-title,
+  .welcome-subtitle {
+    align-self: center;
+    margin: 0;
+    @media screen and (max-width: $responsive-breakpoint) {
+      text-align: center;
+    }
+  }
+
+  .welcome-title {
+    color: $trailhead-purple-80;
+    font-size: 46px;
+    line-height: 62px;
+  }
+
+  .welcome-subtitle {
+    color: $trailhead-violet;
+    font-size: 20px;
+    line-height: 27px;
+  }
+
+  .container {
+    display: flex;
+    align-self: center;
+    padding: 40px 0;
+
+    @media screen and (max-width: $responsive-breakpoint) {
+      flex-direction: column;
+      width: $responsive-width;
+      text-align: center;
+    }
+  }
+
+  .fullpage-left-section {
+    position: relative;
+    width: 460px;
+    margin-inline-start: 10px;
+    margin-inline-end: 2px;
+    font-size: 18px;
+    line-height: 30px;
+
+    @media screen and (max-width: $responsive-breakpoint) {
+      width: $responsive-width;
+    }
+
+    .fullpage-left-content {
+      color: $grey-60;
+      display: inline;
+      margin: 0;
+      margin-inline-end: 2px;
+    }
+
+    .fullpage-left-link {
+      color: $blue-60;
+      display: block;
+      text-decoration: underline;
+      margin-bottom: 30px;
+
+      &:hover,
+      &:active,
+      &:focus {
+        color: $blue-60;
+      }
+    }
+
+    .fullpage-left-title {
+      margin: 0;
+      color: $trailhead-purple-80;
+      font-size: 36px;
+      line-height: 48px;
+    }
+
+    .fx-systems-icons {
+      height: 33px;
+      display: block;
+      background-image: url('#{$image-path}trailhead/firefox-systems.png');
+      margin-bottom: 20px;
+    }
+  }
+
+  .fullpage-form {
+    position: relative;
+    width: 286px;
+    text-align: center;
+    margin-inline-start: 35px;
+    margin-top: 68px;
+
+    h3 {
+      display: none;
+    }
+
+    .fxa-terms {
+      color: $grey-50;
+      font-size: 11px;
+      margin: 4px 0 20px;
+
+      a {
+        color: $grey-50;
+        font-size: 11px;
+      }
+    }
+
+    h3 + p {
+      color: $trailhead-purple-80;
+      font-size: 18px;
+      line-height: 24px;
+    }
+
+    input {
+      background: $white;
+      border: 1px solid $grey-30;
+      border-radius: 2px;
+
+      &:hover {
+        border-color: $grey-50;
+      }
+
+      &.invalid {
+        border-color: $red-60;
+      }
+    }
+
+    button {
+      color: $white;
+      background: $trailhead-violet-70;
+      font-size: 16px;
+      padding: 10px;
+      line-height: 21px;
+
+      &:hover,
+      &:focus {
+        background-color: $trailhead-violet;
+      }
+
+      &:focus {
+        outline: dotted 1px $grey-50;
+      }
+
+      &:active {
+        background-color: $trailhead-violet;
+      }
+    }
+  }
+
+  .section-divider::after {
+    content: '';
+    display: block;
+    border-bottom: 0.5px solid $grey-40;
```

Comment:
This is too dark; I think you probably want `--newtab-border-secondary-color` but confirm with aaron